CenterForOpenScience/osf.io
# [feature] Remove private information from pointer logs [OSF-8607]

## Purpose
Pointer logs show titles of private projects to people who should not have access to that information.


## Changes
Fix the api to remove this information.
Update log parser to handle the lack of information more gracefully.

Before (No view access project):
<img width="540" alt="screen shot 2017-09-11 at 11 13 29 am" src="https://user-images.githubusercontent.com/1322421/30282189-8c0e26dc-96e2-11e7-9005-fa0f40e7a693.png">
After:
<img width="551" alt="screen shot 2017-09-11 at 11 15 02 am" src="https://user-images.githubusercontent.com/1322421/30282197-92b6bf12-96e2-11e7-8c7c-b4e4db817788.png">




## Side effects
NA



## Ticket

[OSF-8607](https://openscience.atlassian.net/browse/OSF-8607)

## QA Notes
1. Deleted project should show up as "a project"
2. Projects you don't have access to will show up as "a project"
3. Projects you do have access to will show up with their current name

Check various iterations of this (registrations, forks, different category types, etc) to see if any bad syntax appears, (e.g., "added a link to project a project" was one I aw locally a bunch)
